src/include/rbd/librbd.h

```diff
@@ -758,6 +768,15 @@ CEPH_RBD_API void rbd_group_image_status_cleanup(
 CEPH_RBD_API void rbd_group_image_status_list_cleanup(
 					      rbd_group_image_status_t *images,
 					      size_t len);
+CEPH_RBD_API int rbd_group_snap_create(rados_ioctx_t group_p, const char *group_name,
+					const char *snap_name);
+CEPH_RBD_API int rbd_group_snap_remove(rados_ioctx_t group_p, const char *group_name,
+					const char *snap_name);
+CEPH_RBD_API int rbd_group_snap_list(rados_ioctx_t group_p, const char *group_name,
+				     rbd_group_snap_spec_t *snaps, size_t *snaps_size);
+CEPH_RBD_API void rbd_group_snap_spec_cleanup(rbd_group_snap_spec_t *group_snap_spec);
```

Comment:
You mean it shouldn't be exposed as a public API? This method is used by rbd_group_snap_spec_list_cleanup. Python bindings only use rbd_group_snap_spec_list_cleanup, I just thought I would add this method for consistency. If we can cleanup a list of snapshot specs, we should be able to cleanup an individual snapshot spec. Do you still want me to remove this method?